index.js

```diff
@@ -69,6 +69,9 @@ function fetchPageOfDataAndFilter(url) {
                         if (parsed && parsed.next && parsed.next.url) {
                             fetchPageOfDataAndFilter(parsed.next.url).then((newEvents) => {
                                 return resolve(importantEvents.concat(newEvents));
+                            })
+                            .catch((err) => {
+                              console.log("Error fetching page of data and filter:", err);
```

Comment:
2 things:

please change "Error fetching page of data and filter:" to `Error fetching page of data for ${parsed.next.url}`

Also, please rethrow the error. (I want to rethrow this error so that the program will stop running if this error occurs.)

Reply:
@danisyellis thanks, I've addressed feedback. Could you please have a look when you have a free sec?